i18n/de.json

```diff
@@ -28,7 +28,9 @@
   "UIKit.MoneyField.highPrecision": "",
   "UIKit.MultilineTextInput.collapse": "Ausblenden",
   "UIKit.MultilineTextInput.expand": "Erweitern",
+  "UIKit.PasswordField.hide": "",
+  "UIKit.PasswordField.show": "",
```

Comment:
@tdeekens can you provide the german translations?

Reply:
show: anzeigen
hide: verbergen

German has many words for hiding but I think "verbergen" is the best. Better then verstecken cause verbergen means it's in sight but not fully visible.